sycl/test/plugins/sycl-targets-order.cpp

```diff
@@ -0,0 +1,68 @@
+// RUN: %clangxx -fsycl -fsycl-targets=spir64-unknown-unknown-sycldevice,nvptx64-unknown-unknown-sycldevice %s -o %t-spir64-nvptx64.out
+// RUN: env SYCL_BE=PI_OPENCL %t-spir64-nvptx64.out
+// RUN: env SYCL_BE=PI_CUDA   %t-spir64-nvptx64.out
+// RUN: %clangxx -fsycl -fsycl-targets=nvptx64-unknown-unknown-sycldevice,spir64-unknown-unknown-sycldevice %s -o %t-nvptx64-spir64.out
+// RUN: env SYCL_BE=PI_OPENCL %t-nvptx64-spir64.out
+// RUN: env SYCL_BE=PI_CUDA   %t-nvptx64-spir64.out
+
+// REQUIRES: opencl, cuda
+
+//==------- sycl-targets-order.cpp - SYCL -fsycl-targets order test --------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+#include <iostream>
+
+int main(int argc, char **argv) {
+
+  // select the default SYCL device
+  cl::sycl::device device{cl::sycl::default_selector{}};
+  std::cout << "Running on SYCL device " << device.get_info<cl::sycl::info::device::name>()
+            << ", driver version " << device.get_info<cl::sycl::info::device::driver_version>()
+            << std::endl;
+
+  // create a queue
+  cl::sycl::queue queue{device};
+
+  // create a buffer of 4 ints to be used inside the kernel code
+  cl::sycl::buffer<cl::sycl::cl_int, 1> buffer(4);
```

Comment:
Any particular reason not just use `int` instead ?

Reply:
I took this from the simple application in [GetStartedWithSYCLCompiler.md](https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedWithSYCLCompiler.md#run-simple-sycl-application).

Sure, I can change it to `int` and fix the cast above.